wpcom-vip-two-factor/sms-provider.php

```diff
@@ -87,8 +87,9 @@ public function delete_token( $user_id ) {
 	public function generate_and_send_token( $user ) {
 		require_once( WPMU_PLUGIN_DIR . '/lib/sms.php' );
 		$token = $this->generate_token( $user->ID ); // Store in variable to prevent generating code twice
-		$message = $token . ' is your ' . $_SERVER['HTTP_HOST'] . " verification code.\r\n" .
-		'@' . $_SERVER['HTTP_HOST'] . ' #' . $token;
+		$hostname = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : ''; 
+		$message = $token . ' is your ' . $hostname . " verification code.\r\n" .
+		'@' . $hostname . ' #' . $token;
```

Comment:
`$hostname` should probably be the value of `home_url()` here (minus the protocol).

Reply:
Ah, forgot that the home url could be different! Thank you for the catch!